website/hugo.toml

```diff
@@ -4,7 +4,7 @@ title = 'Enterprise Contract'
 theme = 'github.com/basil/antora-default-ui-hugo-theme'
 publishDir = '../public'
 copyright = "Red Hat, Inc. All rights reserved."
-relativeURLs = true
+relativeURLs = false
```

Comment:
We want the relative URLs, breaks local preview otherwise

Reply:
ack. I was setting this based on https://gohugo.io/content-management/urls/#relative-urls

Reply:
Works nice for the Camel website, has the side effect of enabling link checking without going over network...